code/API_definitions/carrier_billing.yaml

```diff
@@ -181,12 +187,25 @@ paths:
         - Payment
       summary: Get a list of payments
       operationId: retrievePayments
-      description: Retrieve a list of payments and their details based on some filtering criteria.
+      description: | 
+        Retrieve a list of payments and their details based on some filtering criteria.
+        Regardless the payment criteria provided, response MUST be always limited to payments performed by the API client (i.e same oAuth credentials) triggering this request.
+        This is to guarantee no API client can check payments performed by other, therefore avoiding any legal or privacy topic.
+
+        Considerations regarding `startDate`, `endDate`: 
+        - If both included, return payments in that date range
+        - If no one included, no filtering by date range is applied
+        - If only settled `startDate`, `endDate` is considered current date-time
+        - If only settled `endDate`, `startDate` is considered one hour before
```

Comment:
Humm... I'm not completely comfortable with this by default rule which seems to me a bit arbitrary and not easy to get for developer. Why one hour and not on day or one month? for me we should retrieve all payment which paymentDate is before provided endDate This is seems to me more consistent with the rule when neither date are provided